packages/web-api-scan-runner/src/runner/runner.spec.ts

```diff
@@ -240,6 +232,10 @@ describe(Runner, () => {
 
         setupUpdateScanRunResultCall(getFailingJobStateScanResult(unscannableAxeScanResults.error));
 
+        telemetryManagerMock.setup((t) => t.trackScanStarted(scanSubmittedDate)).verifiable();
+        telemetryManagerMock.setup((t) => t.trackScanTaskFailed()).verifiable(Times.never());
```

Comment:
Have you considered using Mock.Behavior.Strict when this object is created? That way, TypeMoq will throw an error whenever a function that isn't mocked is called. I use it whenever I can because it is easy to forget that a certain function isn't expected to be called, (i.e., it's more future proof) There are times when you can't use it though, and that's reasonable too.